docs/source/faq.rst

```diff
@@ -156,20 +155,37 @@ For example, you can save SVM models trained in the objective function as follow
         clf = sklearn.svm.SVC(C=svc_c)
         clf.fit(X_train, y_train)
 
-        # Save a trained model to a file.
-        with open("{}.pickle".format(trial.number), "wb") as fout:
+        # Save the model using ArtifactStore
+        with open("model.pickle", "wb") as fout:
             pickle.dump(clf, fout)
+        upload_artifact(
```

Comment:
We should get the `artifact_id`  to set the user attributes. In addition, we should follow the convention of this example to use `optuna.XXX`.
```suggestion
        artifact_id = optuna.artifacts.upload_artifact(
```